### PR TITLE
DG: remove need to open PR for small issues

### PR DESCRIPTION
**Pull request by gerhean**
As discussed in slack group.

```
The DG suggests that before making a PR, contributors should post an
issue first and wait for it to be acknowledged.

This is deemed to be unnecessary for small changes, such as fixing
grammatical errors in documentation or simple bug fixes.

Let's add a line in the DG that it is unnecessary to create an issue
for simple documentation fixes or tasks with clear instructions.
```

### CODE REVIEW

File: docs/dg/workflow.md [hunk 1]
```diff
@@ -18,6 +18,7 @@ Our workflow is mostly based on the guidelines given at se-education.org/guides.
 * As we squash the commits when merging a PR, there is ==no need to follow a strict commit organization or write elaborate commit messages for each commit==.<br>
   However, when pushing new commits to your PR branch, do clean up _new_ commits (i.e., commits not yet pushed) e.g., delete temporary print statements added for debugging purposes.
 * In the PR description, please propose a commit message to be used when the PR is merged eventually. The commit message should follow the guidelines given [here](https://se-education.org/guides/guidelines/PRs.html). You may refer to [this PR](https://github.com/reposense/RepoSense/pull/1057) for an example.
+*  For simple documentation fixes or tasks with clear instructions, it is not necessary to create an issue before creating a PR.
```
Comment (Tejas2805) [inline docs/dg/workflow.md:21]:
```suggestion
*  For simple documentation fixes or tasks with clear instructions, it is unnecessary to create an issue before creating a PR.
```

**Review by Tejas2805 — CHANGES_REQUESTED** [review]
Left a comment.

**Comment by gerhean** [comment]
@Tejas2805 Thanks, committed.

**Review by 0blivious — APPROVED** [review]
Pls change the PR title and commit msg accordingly

**Review by 0blivious — APPROVED** [review]

**Comment by Tejas2805** [comment]
@gerhean Your commit messages crosses the 70 characters in a line guideline we follow. Please update it and ask for review again. Rest all LGTM!

**Review by 0blivious — APPROVED** [review]

**Review by Tejas2805 — APPROVED** [review]

File: docs/dg/workflow.md [hunk 2]
```diff
@@ -18,6 +18,7 @@ Our workflow is mostly based on the guidelines given at se-education.org/guides.
 * As we squash the commits when merging a PR, there is ==no need to follow a strict commit organization or write elaborate commit messages for each commit==.<br>
   However, when pushing new commits to your PR branch, do clean up _new_ commits (i.e., commits not yet pushed) e.g., delete temporary print statements added for debugging purposes.
 * In the PR description, please propose a commit message to be used when the PR is merged eventually. The commit message should follow the guidelines given [here](https://se-education.org/guides/guidelines/PRs.html). You may refer to [this PR](https://github.com/reposense/RepoSense/pull/1057) for an example.
+*  For simple documentation fixes or tasks with clear instructions, it is unnecessary to create an issue before creating a PR.
```
Comment (fzdy1914) [inline docs/dg/workflow.md:21]: Weird extra space before the sentence.

Changed message:
```
The DG suggests that before making a PR, contributors should post an 
issue first and wait for it to be acknowledged.

This is deemed to be unnecessary for small changes, such as fixing 
grammatical errors in documentation or simple bug fixes.

Let's add a line in the DG that it is unnecessary to create an issue 
for simple documentation fixes or tasks with clear instructions.
```

**Review by fzdy1914 — CHANGES_REQUESTED** [review]

File: docs/dg/workflow.md (hunk 2 above)
Comment (gerhean) [reply docs/dg/workflow.md]: Done

**Review by fzdy1914 — APPROVED** [review]

**Pull request merged**